dotcom-rendering/src/components/MostPopularFooterGrid.tsx

```diff
@@ -176,7 +177,14 @@ export const MostPopularFooterGrid = ({
 				>
 					<h3 css={titleStyle}>{deeplyRead.heading}</h3>
 					<div css={descriptionStyle}>
-						What readers are spending time with
+						What readers are spending time with (
+						<Link
+							css={descriptionStyle}
```

Comment:
Why do we need these styles?

Reply:
To make sure the link is the same as the rest of the sentence. I think the colour and text size are the only ones we need explicitly but I used the entire group because it's easier to keep in sync. They should never differ.